Context/Context.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Context;
+
+/**
+ * {@inheritdoc}
+ *
+ * @author Ener-Getick <egetick@gmail.com>
+ */
+class Context implements ContextInterface, GroupableContextInterface, VersionableContextInterface, MaxDepthContextInterface, SerializeNullContextInterface
```

Comment:
The goal was [to make it more flexible](https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1074#discussion_r33039525) for a potential 3rd serializer but I agree that we can just not consider the unsupported fields and have a single class.